src/Contract/Test/Testnet.purs

```diff
@@ -26,13 +26,13 @@ import Ctl.Internal.Testnet.Contract
   , runTestnetTestPlan
   , testTestnetContracts
   ) as X
-import Ctl.Internal.Testnet.Types (Era(Babbage), TestnetConfig)
+import Ctl.Internal.Testnet.Types (Era(Babbage), TestnetConfig) as X
```

Comment:
I think, if the goal is to re-export, you should export the other constructors too. In general, it's best to avoid using `X.` imports in the module (OK to import the same module  unqualified)